app/services/github/oauth_client.rb

```diff
@@ -115,13 +115,13 @@ def faraday_middleware_stack
       # <https://github.com/octokit/octokit.rb#caching>
       # and <https://github.com/octokit/octokit.rb/blob/master/lib/octokit/default.rb>
       Faraday::RackBuilder.new do |builder|
-        builder.use Faraday::Request::Retry, exceptions: [Octokit::ServerError]
+        builder.use Faraday::Retry::Middleware, exceptions: [Octokit::ServerError]
         builder.use Octokit::Middleware::FollowRedirects
         builder.use Octokit::Response::RaiseError
         builder.use Octokit::Response::FeedParser
 
         builder.response :logger if Rails.env.development?
-        builder.adapter :patron
+        builder.adapter Faraday.default_adapter
```

Comment:
I'm changing this to default because it's the example provided by Octokit doc

We initially chose Patron to match the tool we were using on Elasticsearch. We no longer use that.